Documentation/network/servicemesh/default-deny-ingress-policy.rst

```diff
@@ -0,0 +1,69 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_default_deny_ingress_policy:
+
+***************************
+Default Deny Ingress Policy
+***************************
+
+Let's apply clusterwide network policy to deny all the traffic by default.
+
+.. literalinclude:: ../../../examples/kubernetes/servicemesh/policy/default-deny.yaml
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/policy/default-deny.yaml
+
+With this policy applied, the request to ``/details`` endpoint will be denied for external and in-cluster traffic.
+
+.. code-block:: shell-session
+
+    $ curl --fail -v http://"$HTTP_INGRESS"/details/1
+    *   Trying 172.19.255.194:80...
+    * Connected to 172.19.255.194 (172.19.255.194) port 80
+    > GET /details/1 HTTP/1.1
+    > Host: 172.19.255.194
+    > User-Agent: curl/8.6.0
+    > Accept: */*
+    >
+    < HTTP/1.1 403 Forbidden
+    < content-length: 15
+    < content-type: text/plain
+    < date: Sun, 17 Mar 2024 13:52:38 GMT
+    < server: envoy
+    * The requested URL returned error: 403
+    * Closing connection
+    curl: (22) The requested URL returned error: 403
+
+    # Capture hubble flows in another terminal
+    $ kubectl --namespace=kube-system exec -i -t cilium-xjl4x -- hubble observe -f --identity ingress
+    Defaulted container "cilium-agent" out of: cilium-agent, config (init), mount-cgroup (init), apply-sysctl-overwrites (init), mount-bpf-fs (init), wait-for-node-init (init), clean-cilium-state (init), install-cni-binaries (init)
+    Mar 17 13:56:00.709: 172.19.0.1:34104 (ingress) -> default/cilium-ingress-basic-ingress:80 (world) http-request DROPPED (HTTP/1.1 GET http://172.19.255.194/details/1)
+    Mar 17 13:56:00.709: 172.19.0.1:34104 (ingress) <- default/cilium-ingress-basic-ingress:80 (world) http-response FORWARDED (HTTP/1.1 403 0ms (GET http://172.19.255.194/details/1))
+
+Let's check if the in-cluster traffic to the same endpoint is being denied:
+
+.. parsed-literal::
+
+    # The test-application.yaml contains a client pod with curl available
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/envoy/test-application.yaml
+    $ kubectl exec -it deployment/client -- curl -s http://$HTTP_INGRESS/details/1
+    Access denied
+
```

Comment:
Is there a paragraph missing here?

Reply:
Yes, I think we should explain what the `allow-ingress-cluster.yaml` file actually does (ie allow traffic from the `ingress` identity to identities inside the cluster.) This might also need some explanation about what the `ingress` identity actually is.